service/version_bundles.go

```diff
@@ -158,20 +159,31 @@ func NewVersionBundles() []versionbundle.Bundle {
 		},
 		{
 			Changelogs: []versionbundle.Changelog{
+				{
+
+					Component:   "CoreOS",
+					Description: "Updated CoreOS version to " + keyv4.CoreosVersion,
+					Kind:        "changed",
+				},
```

Comment:
Some policy it's like that in azure-operator https://github.com/giantswarm/azure-operator/blob/a6cad0dccf66faf7260f4b28ca87acb058eb79cc/service/version_bundles.go#L33-L37

- Components are lowercased.

- It's "containerlinux", not "coreos"
- Full stop in the description.
- I wouldn't import the `keyv4.CoreosVersion` (import cycles and stuff) constant here and just type it in instead.

This stuff is important from ux perspective.